backup/src/main/java/io/camunda/zeebe/backup/processing/state/CheckpointState.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.processing.state;
+
+public interface CheckpointState {
+
+  long NO_CHECKPOINT = 0L;
```

Comment:
:wrench: :question: Is there any specific reason you chose 0 and not -1? I'd prefer -1 because it's a bit more obvious that this is an "invalid" value.